distribution/lib/Standard/Base/0.0.0-dev/src/Data/Map.enso

```diff
@@ -141,7 +141,7 @@ type Map key value
     remove : Any -> Map ! No_Such_Key
     remove self key =
         Panic.catch Any (self.remove_builtin key) _->
-            Error.throw No_Such_Key.Error self key
+            Error.throw (No_Such_Key.Error self key)
```

Comment:
That such errors can slip through makes me worry about our test coverage. We should have better tests for Map :/